client/public.go

```diff
@@ -232,7 +232,7 @@ func (p *Profile) CreateProfile(ctx context.Context) error {
 			errMap := ErrorMap{}
 			for _, f := range errResp.Fields {
 				fn := strings.Split(f.Name, ":")
-				errMap.AppendErrorField(fn[1], f.Message)
+				errMap.AppendErrorField(fn[0], f.Message)
```

Comment:
This should really be
```golang
			errMap.AppendErrorField(fn[len(fn)-1], f.Message)
```
Sometimes the error comes in the form of `parent_element:field`, and all we really want is the field name.  Which means we should be grabbing the last element of the slice, not the first.